Dockerfile

```diff
@@ -51,7 +169,7 @@ RUN --mount=type=cache,target=/root/.cache/go-build \
         && GOPATH="$GOPATH/src/github.com/docker/distribution/Godeps/_workspace:$GOPATH" \
            go build -buildmode=pie -o /build/registry-v2 github.com/docker/distribution/cmd/registry \
         && case $(dpkg --print-architecture) in \
-               amd64|ppc64*|s390x) \
+               amd64|armhf|ppc64*|s390x) \
```

Comment:
I can't understand why this is change in this commit

Reply:
Moved to https://github.com/moby/moby/pull/40362, and rebased this PR on top of that